src/Employer/Employer.Web/Controllers/EmployerVacancyController.cs

```diff
@@ -9,6 +10,7 @@ public class EmployerVacancyController : Controller
         [HttpGet("employer-create-vacancy", Name = RouteNames.EmployerCreateVacancy_Get)]        
         public IActionResult CreateVacancy()
         {
+            TempData[TempDataKeys.ReferredFromMAHome] = "Return to home";
```

Comment:
This is weird. Why should this controller be setting the content of a link on another page?  This should be more of a boolean flag and then the page inspecting it then decides how to handle this information.